eng/common/scripts/modules/ChangeLog-Operations.psm1

```diff
@@ -94,8 +95,8 @@ function Confirm-ChangeLogEntry {
   }
 
   if ($ForRelease -eq $True) {
-    $CurrentDate = Get-Date -Format "yyyy-MM-dd"
-    if ($changeLogEntry.ReleaseStatus -ne "($CurrentDate)") {
+    $ExpectedStatus = '(' + $BuildNo.Substring(0,4) + '-' + $BuildNo.Substring(4,2) + '-' + $BuildNo.Substring(6,2) + ')'
```

Comment:
We are making an assumption about the format of BuildNo that isn't always true. 

I think perhaps we should just relax this check and only verify that the ReleaseStatus is a date and not "(Unreleased)".

Reply:
Since the $RELEASE_TITLE_REGEX already matched date or Unreleased all that needed is to make sure its not Unreleased at this point.